src/main/java/cc/polyfrost/oneconfig/renderer/TextRenderer.java

```diff
@@ -1,243 +0,0 @@
-/*
- * This file is part of OneConfig.
- * OneConfig - Next Generation Config Library for Minecraft: Java Edition
- * Copyright (C) 2021, 2022 Polyfrost.
- *   <https://polyfrost.cc> <https://github.com/Polyfrost/>
- *
- * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
- *
- *   OneConfig is licensed under the terms of version 3 of the GNU Lesser
- * General Public License as published by the Free Software Foundation, AND
- * under the Additional Terms Applicable to OneConfig, as published by Polyfrost,
- * either version 1.0 of the Additional Terms, or (at your option) any later
- * version.
- *
- *   This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
- * Lesser General Public License for more details.
- *
- *   You should have received a copy of the GNU Lesser General Public
- * License.  If not, see <https://www.gnu.org/licenses/>. You should
- * have also received a copy of the Additional Terms Applicable
- * to OneConfig, as published by Polyfrost. If not, see
- * <https://polyfrost.cc/legal/oneconfig/additional-terms>
- */
-
-package cc.polyfrost.oneconfig.renderer;
-
-import cc.polyfrost.oneconfig.internal.assets.Colors;
-import cc.polyfrost.oneconfig.renderer.font.Font;
-import cc.polyfrost.oneconfig.utils.InputHandler;
-import cc.polyfrost.oneconfig.utils.NetworkUtils;
-import com.google.common.annotations.Beta;
-import org.lwjgl.nanovg.NVGColor;
-
-import java.util.ArrayList;
-
-import static cc.polyfrost.oneconfig.renderer.RenderManager.color;
-import static org.lwjgl.nanovg.NanoVG.*;
-
-public class TextRenderer {
```

Comment:
also will need to be changed w/ polyui
![banner](https://user-images.githubusercontent.com/66741050/202937938-471f8822-31d0-4f84-b9e1-d30df69f6760.jpg)